cypress/integration/pages/onDemandRadioEpisode/tests.js

```diff
@@ -0,0 +1,64 @@
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
+  describe(`Tests for ${service} ${pageType}`, () => {
+    describe('On Demand Radio body', () => {
+      it('should render a H1, which contains/displays a styled headline', () => {
+        cy.request(`${Cypress.env('currentPath')}.json?renderer_env=live`).then(
+          ({ body }) => {
+            cy.get('h1').should('contain', body.promo.headlines.headline);
+          },
+        );
+      });
+
+      it('should render a paragraph, which contains/displays a styled summary', () => {
```

Comment:
We can remove this, as it has been implemented as an integration test: https://github.com/bbc/simorgh/blob/22d01e907c47c6b6180464a722099dc991420f0f/src/integration/pages/onDemandRadioEpisode/commonEpisodeTests.js#L18

Reply:
Removed